Lib/test/test_dis.py

```diff
@@ -1617,197 +1616,197 @@ def _prepare_test_cases():
 Instruction = dis.Instruction
 
 expected_opinfo_outer = [
```

Comment:
This test is just preposterous. Let's commit to doing this in a different way in a future PR.

Reply:
@markshannon loves to hate this test too.